TeachingRecordSystem/src/TeachingRecordSystem.AuthorizeAccess/Infrastructure/Security/OneLoginAuthenticationSchemeProvider.cs

```diff
@@ -155,15 +155,6 @@ void IConfigureNamedOptions<OneLoginOptions>.Configure(string? name, OneLoginOpt
 
         options.SignInScheme = AuthenticationSchemes.FormFlowJourney;
 
-        options.Events.OnRedirectToIdentityProvider = context =>
```

Comment:
This is no longer required since a) One Login has increased the maximum size of URLs they accept and 2) we're no longer passing `AuthenticationProperties` along.